frontend/__mocks__/crds.ts

```diff
@@ -15,16 +15,23 @@ export const testCRD: CustomResourceDefinitionKind = {
   },
   spec: {
     group: 'testapp.coreos.com',
-    version: 'v1alpha1',
-    validation: {
-      openAPIV3Schema: {},
-    },
+    versions: [
+      {
+        name: 'v1',
```

Comment:
I believe this can remain 'v1alpha1' since it's actually the version of the custom resource, not the CRD api version.